src/commands/automercy.tsx

```diff
@@ -1,9 +1,21 @@
+import * as elements from "typed-html";
 import z from "zod";
 import { createCommand } from "../command/createCommand";
-import { getStickMessage } from "../deleteMessage";
-import { Message, PartialMessage } from "discord.js";
+import { updateSite } from "../web/server";
 
-let autoMercy: boolean = false;
+export let autoMercy: boolean = false;
+
+export const toggleAutoMercy = () => {
+  autoMercy = !autoMercy;
+};
+
+export const WebAutoMercyDisplay = () => {
```

Comment:
This is just like a react component. You can tell that its a component because it starts with a capital letter. All components just return HTML.